velox/expression/tests/SignatureBinderTest.cpp

```diff
@@ -688,3 +689,14 @@ TEST(SignatureBinderTest, customType) {
           .build(),
       "Type doesn't exist: FANCY_TYPE");
 }
+
+TEST(SignatureBinderTest, hugeIntType) {
+  // Logical type as an argument type.
+  {
+    auto signature = exec::FunctionSignatureBuilder()
+                         .returnType("hugeint")
+                         .argumentType("hugeint")
+                         .build();
+    testSignatureBinder(signature, {HUGEINT()}, HUGEINT());
```

Comment:
Thank you for adding this test! Could you also test that if a function signature is T -> T, whether {HUGEINT()} argument type can be correctly resolved?

Reply:
added